tests/robot-tests/tests/admin_and_public/bau/prerelease_visibility.robot

```diff
@@ -29,22 +29,47 @@ Verify release summary
 Upload subject
     user uploads subject    UI test subject    upload-file-test.csv    upload-file-test.meta.csv
 
-Go to 'Sign Off' page
+Check release isn't publically visible
     user clicks link    Sign off
     user waits for page to finish loading
     user waits until page contains element    testid:public-release-url
-    ${PUBLIC_RELEASE_LINK}=    Get Value    xpath://*[@data-testid="public-release-url"]
+    ${PUBLIC_RELEASE_LINK}=    Get Value    testid:public-release-url
     check that variable is not empty    PUBLIC_RELEASE_LINK    ${PUBLIC_RELEASE_LINK}
     Set Suite Variable    ${PUBLIC_RELEASE_LINK}
 
-Go to Public Release Link
     user navigates to public frontend    ${PUBLIC_RELEASE_LINK}
     user waits until page contains    Page not found
     user checks page does not contain    ${RELEASE_NAME}
 
 Return to admin
     user navigates to admin dashboard    Bau1
 
+Create methodology for release
+    user creates methodology for publication    ${PUBLICATION_NAME}
+
+    user clicks link    Manage content
+    user creates new content section    1    Methodology content section 1
+    ...    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+    user adds text block to editable accordion section    Methodology content section 1
+    ...    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+    user adds content to accordion section text block    Methodology content section 1    1
+    ...    Content 1    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+
+Set methodology to be published alongside release
+    approve methodology from methodology view    WithRelease    ${PUBLICATION_NAME} - ${RELEASE_NAME}
+
+Check methodology isn't publically visible
+    user waits until page contains element    testid:public-methodology-url
+    ${PUBLIC_METHODOLOGY_LINK}=    Get Value    testid:public-methodology-url
+    check that variable is not empty    PUBLIC_METHODOLOGY_LINK    ${PUBLIC_METHODOLOGY_LINK}
+    Set Suite Variable    ${PUBLIC_METHODOLOGY_LINK}
+
+    user navigates to public frontend    ${PUBLIC_METHODOLOGY_LINK}
+    user waits until page contains    Page not found
```

Comment:
Would it make sense to check it's also not visible on the public methodologies page as well after this?